addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3020,6 +3020,10 @@ export class OdooEditor extends EventTarget {
                     this.toolbar.style.visibility = show ? 'visible' : 'hidden';
                 }
                 if (show === false) {
+                    this.toolbar.querySelectorAll('.dropdown-menu.show').forEach((e)=>{
+                        const dropdownButton = e.parentElement.querySelector('[data-bs-toggle="dropdown"]');
+                        dropdownButton && dropdownButton.click();
```

Comment:
@Zynton , can I do something like this or we prefer using an if statement ?

Reply:
We're in 16.4 so we can use `dropdownButton?.click();`.